### DISCUSSION TITLE
Mic Level Indicator?

### DISCUSSION THREAD

**Discussion by jkrauska** [tphakala/birdnet-go#333] [General]
There's a mic level indicator in the dev release which DOES seem to help show the current levels, but I don't know how to interpret it.

If it's at 50% is that "high" or low?

<img width="94" alt="Screenshot 2024-12-19 at 1 47 38 PM" src="https://github.com/user-attachments/assets/6c217af8-5f9b-4b8b-a3ec-eecf807e9869" />

When I dial it up, it does turn red at 100%. I guess that means clipping?

**Comment by tphakala** [comment]
Level is fine as long as it is not 0 or clipping which is indicated by meter turning to red as you guessed. Very low levels mean low volume on audio clips though, and currently there is no way to boost clip volume within BirdNET-Go. 

Levels are perfect when audio is clearly visualised in spectrogram, like here 

![image](https://github.com/user-attachments/assets/dc0ac6a2-1332-4818-9ea8-e16cca90d327)
